### PR TITLE
Old Glast Heim and Nightmare Glast Heim

### PR DESCRIPTION
**Pull request by Ridley8819**
Main Content: Updated Old Glast Heim Instance
                        Added Nightmare Glast Heim Dungon (this is not OGH hard mode, it's a dungeon)

- added quest_db entries
- added and updated mobs, skills and items
- Updated dialogues to official
https://www.youtube.com/watch?v=MzcJqCJZxzY
- Added H.U.L.D.
- Added Temporal Boots Enchants (did my best to bring it close to official, basic enchanter were taken from idAthena)
--> Trade 1 Crystal to 1 Pair of Boots
--> Once Refined to at least +7 you can change Temporal_Boots to Temporal_STAT_Boots (refine remains)
--> You can now enchant your Boots on slot 4 and 3 (coagulated_spell)
--> You can slot socket enchant your Boots anytime, but afterwards you can NOT add any other enchants (refine and current enchants remain) <-- success rate custom
https://www.youtube.com/results?search_query=ogh+enchant
- Added NPC's to trade White_Knightage_Card and Khalitzburg_Knightage_Card  https://forums.warpportal.com/index.php?/topic/171442-new-ogh-boots-enchants-system-question/
- Added Treasure Room (idAthena)
- .MyMobs to 'MyMobs (https://github.com/HerculesWS/Hercules/pull/1325)
- renamed Knight_Cards to Knightage_Cards

---
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1370)

### CODE REVIEW

**Comment by Ridley8819** [comment]
Seems I added jro enchant mechanics, currently talking with ziu and I will adjust it to kro

**Comment by Ridley8819** [comment]
Info about kro enchantment:
1st slot the STAT boots
2nd enchant then with chance of breaking 
3rd apply ransom 3rd slot enchant (guaranteed) 
No +7 refine required

**Comment by Ridley8819** [comment]
Note2: upon slotting you loose all enchants

File: db/re/mob_db.conf [hunk 1]
```diff
@@ -65147,15 +65147,441 @@ mob_db: (
...
+	Id: 2477
+	SpriteName: "NG_WANDER_MAN"
+	Name: "Nightmare Wander Man"
+	Lv: 151
+	Hp: 130682
+	Sp: 1
+	Exp: 19814
+	JExp: 17228
+	AttackRange: 2
+	Attack: [2827, 3045]
```
Comment (Asheraf) [inline db/re/mob_db.conf]: i think for the monster attack is should be [2827, 218]  like other monsters the attack2-attack1 in divine-pride

File: db/re/item_db.conf [hunk 2]
```diff
@@ -73763,6 +73781,38 @@ item_db: (
...
+	Weight: 10
+   BuyingStore: true
+},
+{
+	Id: 6610
+	AegisName: "Glast_Horrendous_Mouth"
+	Name: "Glast Horrendous Mouth"
+	Buy: 390
+	Weight: 10
+   BuyingStore: true
```
Comment (Asheraf) [inline db/re/item_db.conf]: tab :P

File: db/re/item_db.conf [hunk 3]
```diff
@@ -73763,6 +73781,38 @@ item_db: (
 	Weight: 10
 },
 {
+	Id: 6609
+	AegisName: "Glast_Decayed_Nail"
+	Name: "Glast Decayed Nail"
+	Buy: 82
+	Weight: 10
+   BuyingStore: true
```
Comment (Asheraf) [inline db/re/item_db.conf]: tab :P

File: db/re/item_db.conf [hunk 4]
```diff
@@ -99640,6 +99697,21 @@ item_db: (
...
+	Id: 13086
+	AegisName: "Goldsmithing_Dagger"
+	Name: "Goldsmithing Dagger"
+	Type: 4
+	Buy: 20
+	Weight: 500
+	Atk: 35
+	Matk: 25
+	Range: 1
+	Job: 0x028352C6
```
Comment (Asheraf) [inline db/re/item_db.conf]: Job: {
		Swordsman: true
		Magician: true
		Merchant: true
		Thief: true
		Soul_Linker: true
		Ninja: true 
	}

File: db/re/item_db.conf [hunk 5]
```diff
@@ -104477,6 +104549,20 @@ item_db: (
...
 },
 {
+	Id: 13440
+	AegisName: "Ceremonial_Sword"
+	Name: "Ceremonial Sword"
+	Type: 4
+	Buy: 20
+	Weight: 500
+	Atk: 60
+	Job: 0x000654E3
```
Comment (Asheraf) [inline db/re/item_db.conf]: Job: {
		Novice: true
		Swordsman: true
		Merchant: true
		Thief: true
		Knight: true
		Blacksmith: true
		Assassin: true
		Crusader: true
		Rogue: true
		Alchemist: true
	}

File: db/re/item_db.conf [hunk 6]
```diff
@@ -123739,6 +123825,19 @@ item_db: (
...
 },
 {
+	Id: 15066
+	AegisName: "Engraved_Armor"
+	Name: "Engraved Armor"
+	Type: 5
+	Weight: 3000
+	Def: 120
+	Slots: 1
+	Job: 0x00004082
```
Comment (Asheraf) [inline db/re/item_db.conf]: Job: {
		Swordsman: true
		Knight: true
		Crusader: true
	}

File: db/re/mob_db.conf (hunk 1 above)
Comment (Ridley8819) [inline db/re/mob_db.conf]: You are right, I just noticed those.

File: db/re/item_db.conf (hunk 6 above)
Comment (Ridley8819) [inline db/re/item_db.conf]: Thank you for those :D

File: db/re/mob_db.conf [hunk 7]
```diff
@@ -65018,25 +65136,452 @@ mob_db: (
...
+	DamageMotion: 240
+	MvpExp: 0
+	Drops: {
+		Transparent_Cloth: 4400
+		Wedding_Veil: 10
+		Manteau_: 5
+		Red_Gemstone: 750
+		Manteau: 10
+		Crystal_Jewel__: 10
+
```
Comment (Asheraf) [inline db/re/mob_db.conf]: extra empty line

File: db/re/mob_db.conf [hunk 8]
```diff
@@ -65018,25 +65136,452 @@ mob_db: (
...
+		ChangeChase: true
+		ChangeTargetMelee: true
+		ChangeTargetChase: true
+	}
+	MoveSpeed: 350
+	AttackDelay: 528
+	AttackMotion: 1000
+	DamageMotion: 396
+	MvpExp: 0
+	Drops: {
```
Comment (Asheraf) [inline db/re/mob_db.conf]: no need for this if it's empty

**Comment by AnisotropicDefixation** [comment]
The "Knightage" thing is just engrish though, you can just check the original Korean card names.

http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=5&ID=4609

http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=5&ID=4608

칼리츠버그 기사단 카드 =  Khalitzburg Knight

백색의 기사단 카드 = White Knight

**Comment by Ridley8819** [comment]
Yea but since I don't know anyone speaking Korean I thought its better to follow the English name

**Comment by Ridley8819** [comment]
Applied all corrections

**Comment by hemagx** [comment]
i stand with @AnisotropicDefixation we should those this names instead

**Comment by Ridley8819** [comment]
Additional References:
http://irowiki.org/wiki/Old_Glast_Heim
http://www.divine-pride.net/database/map/1@gl_k
https://forums.warpportal.com/index.php?/topic/186174-couldnt-enter-ogh-treasure-room/
http://ratemyserver.net/index.php?page=areainfo&area=3065&re_mob=1

**Pull request closed without merging**